annif/project.py

```diff
@@ -188,6 +213,8 @@ def train(self, corpus, backend_params=None):
             corpus.set_subject_index(self.subjects)
         if backend_params is None:
             backend_params = {}
+        if self.transformer is not None:
```

Comment:
Likewise here: no need for a None check if a passthrough transformer fallback is used in the property function